base_view_inheritance_extension/models/ir_ui_view.py

```diff
@@ -4,15 +4,8 @@
 # Copyright 2023 Tecnativa - Carlos Dauden
 # License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
 import ast
-import logging
 import re
 
-try:
-    import astor
```

Comment:
Does this mean that astor can be dropped as a python dependency in the manifest?

Reply:
I believe so. This should be done in 18.0 too then